tests/Resources/keywords_sidlistningsblock.robot

```diff
@@ -6,9 +6,8 @@ Library  SeleniumLibrary
 
 Sidlistningsblock Test
 
-    # [Documentation] Verifies that block is existing and isnt empty
+    # [Documentation] Verifies that block is existing, contains a link and check statusCode.
 
-    Wait Until Element Is Visible               css:.pagelistblock
-    ${elementsinblocks} =                       Get Element Count    css:.pagelistblock
-    Log                                         ${elementsinblocks}
-    Should Be True                              ${elementsinblocks} > 0
+    Wait Until Element Is Visible               css:.pagelistblock li a
+    ${statusCodeResult}                         check status code       ${URL}
```

Comment:
jag tror variabeln `${URL}` syftar på den sida vi testar, alltså inte de länkar som listas i blocket. jag har precis gjort en fix i PCR-testbranchen som går igenom en lista med länkar och testar dem var för sig. Skulle kanske kunna vara något att inspireras av? 

Egentligen borde vi kanske göra en funktion i `keywords_general.robot` där man kan stoppa in en lista med länkar, och returnerar true eller false beroende på om alla länkar får ett bra svar eller ej...

Reply:
Här är länk till filen i PCR-testbranchen: https://github.com/Vastra-Gotalandsregionen/selenium-vgregion_se/blob/ecdface688eb8b3c540fde9dbd6858b8c68dd11b/tests/Resources/keywords_pcr-test.robot

Reply:
Ja, just det. Det gör den ju. Inte konstigt att det blir rätt statuskod. Kollar på den.

Låter som en bra idè. Det kan jag kika på för att lära mig mer :).

Reply:
Uppdaterad länk till kod som fungerar som det är tänkt:
https://github.com/Vastra-Gotalandsregionen/selenium-vgregion_se/blob/feature/pcr-test/tests/Resources/keywords_pcr-test.robot